view/plot-3d/mixin.pug

```diff
@@ -0,0 +1,6 @@
+mixin plot3d()
```

Comment:
If the template is static it is better to use `index.pug` and call it by `include PATH` without mixin